docs/routes.md

```diff
@@ -0,0 +1,39 @@
+# Routes
+
+Each API route is defined in a separate file within a specific directory (by default, `./api`). Here's an example:
+
+```ts
+// api/index.ts
+import { withEdgeSpec } from "../src/with-edge-spec"
+import { z } from "zod"
+
+export default withEdgeSpec({
+  auth: "none",
+  methods: ["POST"],
+  jsonBody: z.object({
+    a: z.number(),
+    b: z.number(),
+  }),
+  jsonResponse: z.object({
+    sum: z.number(),
+  }),
```

Comment:
Might be worth adding some clarification about zod. Not everyone who will read this may be familiar!